torch/nn/init.py

```diff
@@ -345,7 +345,7 @@ def orthogonal_(tensor, gain=1):
         raise ValueError("Only tensors with 2 or more dimensions are supported")
 
     rows = tensor.size(0)
-    cols = tensor.numel() / rows
+    cols = int(tensor.numel() / rows)
```

Comment:
This can be made `tensor.numel() // rows`.

Reply:
I amended the last commit.